feature-libs/my-account/organization/occ/adapters/occ-org-unit.adapter.ts

```diff
@@ -0,0 +1,286 @@
+  }
+
+  updateAddress(
+    userId: string,
+    orgUnitId: string,
+    addressId: string,
+    address: B2BAddress
+  ): Observable<B2BAddress> {
+    return this.http
+      .patch<Occ.B2BAddress>(
+        this.getAddressEndpoint(userId, orgUnitId, addressId),
+        address
+      )
+      .pipe(this.converter.pipeable(B2B_ADDRESS_NORMALIZER));
+  }
+
+  deleteAddress(
+    userId: string,
+    orgUnitId: string,
+    addressId: string
+  ): Observable<any> {
+    return this.http
+      .delete<Occ.B2BAddress>(
+        this.getAddressEndpoint(userId, orgUnitId, addressId)
+      )
+      .pipe(this.converter.pipeable(B2B_ADDRESS_NORMALIZER));
+  }
+
+  protected getOrgUnitEndpoint(userId: string, orgUnitId: string): string {
+    return this.occEndpoints.getUrl('orgUnit', { userId, orgUnitId });
+  }
+
+  protected getOrgUnitsEndpoint(userId: string): string {
+    return this.occEndpoints.getUrl('orgUnits', { userId });
+  }
+
+  protected getAvailableOrgUnitsEndpoint(userId: string): string {
+    return this.occEndpoints.getUrl('orgUnitsAvailable', { userId });
+  }
+
+  protected getOrgUnitsTreeEndpoint(userId: string): string {
+    return this.occEndpoints.getUrl('orgUnitsTree', { userId });
+  }
+
+  protected getOrgUnitsApprovalProcessesEndpoint(userId: string): string {
+    return this.occEndpoints.getUrl('orgUnitsApprovalProcesses', { userId });
+  }
+
+  protected getUsersEndpoint(
+    userId: string,
+    orgUnitId: string,
+    roleId: string,
+    params?: B2BSearchConfig
+  ): string {
+    return this.occEndpoints.getUrl(
+      'orgUnitUsers',
+      {
+        userId,
+        orgUnitId,
+        roleId,
+      },
+      params
+    );
+  }
+
+  protected getRolesEndpoint(
+    userId: string,
+    orgCustomerId: string,
+    params: { roleId: string }
+  ): string {
+    return this.occEndpoints.getUrl(
+      'orgUnitUserRoles',
+      { userId, orgCustomerId },
+      params
+    );
+  }
+
+  protected getRoleEndpoint(
+    userId: string,
+    orgCustomerId: string,
+    roleId: string
+  ): string {
+    return this.occEndpoints.getUrl('orgUnitUserRole', {
+      userId,
+      orgCustomerId,
+      roleId,
+    });
+  }
+
+  protected getApproversEndpoint(
+    userId: string,
+    orgUnitId: string,
+    orgCustomerId: string,
+    params: { roleId: string }
+  ): string {
+    return this.occEndpoints.getUrl(
+      'orgUnitApprovers',
+      { userId, orgUnitId, orgCustomerId },
+      params
+    );
+  }
+
+  protected getApproverEndpoint(
+    userId: string,
+    orgUnitId: string,
+    orgCustomerId: string,
+    roleId: string
+  ): string {
+    return this.occEndpoints.getUrl('orgUnitApprover', {
+      userId,
+      orgUnitId,
+      orgCustomerId,
+      roleId,
+    });
+  }
+
+  protected getAddressesEndpoint(userId: string, orgUnitId: string): string {
+    return this.occEndpoints.getUrl('orgUnitsAddresses', { userId, orgUnitId });
+  }
+
+  protected getAddressEndpoint(
```

Comment:
IMO It looks like an overkill to create separate `protected` method for getting each endpoint. It's already taken from calling a generic `occEndpoints.getUrl()`. But I don't mind leaving it, as it doesn't hurt.
@tobi-or-not-tobi do you like this pattern or should we avoid it in future?

Reply:
Agreed, but because of time we probably leave it now.